### PR TITLE
fix: add missing ad config method to ssr topic

### PR DESCRIPTION
**Pull request by colincclark**

### CODE REVIEW

File: packages/ssr/src/lib/graphql-logging-link.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ class LogLink extends ApolloLink {
     this.uri = uri;
     this.logger = logger;
   }
+
```
Comment (Belema) [inline packages/ssr/src/lib/graphql-logging-link.js]: Do we need this?

File: packages/ssr/src/lib/run-server.js [hunk 2]
```diff
@@ -8,9 +8,9 @@ const { fragmentMatcher } = require("@times-components/schema");
 const { getDataFromTree } = require("react-apollo");
 const { InMemoryCache } = require("apollo-cache-inmemory");
 const ReactDOMServer = require("react-dom/server");
-const safeStringify = require("./safe-stringify");
 const { ServerStyleSheet } = require("styled-components");
 const { ApolloLink } = require("apollo-link");
+const safeStringify = require("./safe-stringify");
```
Comment (Belema) [inline packages/ssr/src/lib/run-server.js]: Why?

File: packages/ssr/src/lib/graphql-logging-link.js (hunk 1 above)
Comment (colincclark) [reply packages/ssr/src/lib/graphql-logging-link.js]: yep, that is what happens when you format that package

File: packages/ssr/src/lib/run-server.js (hunk 2 above)
Comment (colincclark) [reply packages/ssr/src/lib/run-server.js]: formatting

File: packages/ssr/src/lib/graphql-logging-link.js (hunk 1 above)
Comment (Belema) [reply packages/ssr/src/lib/graphql-logging-link.js]: No

Comment (colincclark) [reply packages/ssr/src/lib/graphql-logging-link.js]: it looks better

File: packages/ssr/src/lib/run-server.js (hunk 2 above)
Comment (colincclark) [reply packages/ssr/src/lib/run-server.js]: Changed back, must have been a deps issue

**Pull request merged**